src/taco-utils/helpCommandBase.ts

```diff
@@ -95,9 +95,10 @@ module TacoUtility {
             Logger.log(util.format("   <synopsis>%s %s</synopsis><br/>", this.cliName, "<COMMAND>"));
 
             var nameDescriptionPairs: INameDescription[] = new Array();
-            for (var i in this.commandsFactory.listings) {
-                nameDescriptionPairs.push({ name: i, description: this.commandsFactory.listings[i].description, category: this.commandsFactory.listings[i].categoryTitle });
-            }
+            var listings: any = this.commandsFactory.listings;
+            Object.keys(this.commandsFactory.listings).forEach(function(i: string) {
```

Comment:
you can replace this.commandsFactory.listings here with listings since you already named it.